js/ui/text_box/ui.text_editor.base.js

```diff
@@ -462,6 +473,11 @@ const TextEditorBase = Editor.inherit({
     },
 
     _renderLabel: function() {
+
+        this._$labelContainer = $(this._getLabelContainer()).get(0);
```

Comment:
```suggestion
        this._labelContainerElement = $(this._getLabelContainer()).get(0);
```
This is not an instance of the jQuery or Renderer object